spec/services/prosecution_case_recorder_spec.rb

```diff
@@ -61,5 +61,46 @@
       record
       expect(prosecution_case.reload.body).to eq(body)
     end
+
+    context "when other prosecution case exists" do
+      let!(:other_prosecution_case) do
+        ProsecutionCase.create!(
+          id: SecureRandom.uuid,
+          body:,
+        )
+      end
+
+      context "when this case has a PCDO for the other case's defendant" do
```

Comment:
First time you use PCDO it's good to be explicit, then you can use the acronimo, for example:
"when this case has a PCDO (Prosecution Case Defendant Offence) for the other case's defendant"